docs/content/doc/help/faq.en-us.md

```diff
@@ -72,132 +48,119 @@ https://github.com/loganinak/MigrateGitlabToGogs
+#### Preventing spammers
 
-See the [reverse proxy guide]({{< relref "doc/usage/reverse-proxies.en-us.md" >}}) for a solution with nginx.
+There are multiple things you can combine to prevent spammers.
 
-## Custom Templates not loading or working incorrectly
+1. By whitelisting or blocklisting certain email domains
+2. By only whitelisting certain domains with OpenID (see below)
+3. Setting `ENABLE_CAPTCHA` to `true` in your `app.ini` and properly configuring `RECAPTCHA_SECRET` and `RECAPTCHA_SITEKEY`
+4. Settings `DISABLE_REGISTRATION` to `true` and creating new users via the [CLI]({{< relref "doc/usage/command-line.en-us.md" >}}), [API]({{< relref "doc/developers/api-usage.en-us.md" >}}), or Gitea's Admin UI
 
-Gitea's custom templates must be added to the correct location or Gitea will not find and use them.  
-The correct path for the template(s) will be relative to the `CustomPath`
+#### Only allow/block certain email domains
 
-1. To find `CustomPath`, look for Custom File Root Path in Site Administration -> Configuration
+You can configure `EMAIL_DOMAIN_WHITELIST` or `EMAIL_DOMAIN_BLOCKLIST` in your app.ini under `[service]`
 
-- If that doesn't exist, you can try `echo $GITEA_CUSTOM`
+#### Only allow/block certain OpenID providers
 
-2. If you are still unable to find a path, the default can be [calculated above](#where-does-gitea-store-x-file)
-3. Once you have figured out the correct custom path, you can refer to the [customizing Gitea]({{< relref "doc/advanced/customizing-gitea.en-us.md" >}}) page to add your template to the correct location.
+You can configure `WHITELISTED_URIS` or `BLACKLISTED_URIS` under `[openid]` in your `app.ini`  
+**NOTE:** whitelisted takes precedence, so if it is non-blank then blacklisted is ignored
 
-## Active user vs login prohibited user
+#### Issue only users
 
-In Gitea, an "active" user refers to a user that has activated their account via email.  
-A "login prohibited" user is a user that is not allowed to log in to Gitea anymore
+The current way to achieve this is to create/modify a user with a max repo creation limit of 0.
+For organization repos, teams can be set up to only access the issue tracker.
 
-## Setting up logging
+#### Enable Fail2ban
 
-- [Official Docs]({{< relref "doc/advanced/logging-documentation.en-us.md" >}})
+Use [Fail2Ban]({{< relref "doc/usage/fail2ban-setup.en-us.md" >}}) to monitor and stop automated login attempts or other malicious behavior based on log patterns
 
-## What is Swagger?
+### How to add/use custom themes
 
-[Swagger](https://swagger.io/) is what Gitea uses for its API.  
-All Gitea instances have the built-in API, though it can be disabled by setting `ENABLE_SWAGGER` to `false` in the `api` section of your `app.ini`  
-For more information, refer to Gitea's [API docs]({{< relref "doc/developers/api-usage.en-us.md" >}})
+See [here]({{< relref "doc/advanced/customizing-gitea.en-us.md" >}}#customizing-the-look-of-gitea).
```

Comment:
Generally best to avoid `here` links:

```suggestion
See the [customizing the look of gitea]({{< relref "doc/advanced/customizing-gitea.en-us.md" >}}#customizing-the-look-of-gitea) section of our documentation.
```